Import-VIRole.ps1

```diff
@@ -118,9 +120,16 @@ function Import-VIRole
                 Write-Warning -Message "Permission named $(($_.Exception.Message.Split("'"))[1]) not found"
             }
         }
-
+	if (!$Overwrite)
+	{
```

Comment:
7 instead of 8 here and in the changes below as well. Also lines 127-128 should get another 4 space indent to be under this if block.